sklearn/metrics/classification.py

```diff
@@ -1072,7 +1072,7 @@ def precision_recall_fscore_support(y_true, y_pred, beta=1.0, labels=None,
                 raise ValueError('All labels must be in [0, n labels). '
                                  'Got %d < 0' % np.min(labels))
 
-            y_true = y_true[:, labels[:n_labels]]
+            y_true = y_true[:, :n_labels]
```

Comment:
we still need to index by `labels`. And we need to do the same thing to `y_true` as to `y_pred`